docs/2.5.5/docs/canton/architecture/requirements/requirements.rst

```diff
@@ -1,7 +1,7 @@
 ..
      Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates
 ..
-    
+
```

Comment:
Fun fact: the Canton build actually fails when you make that change in the Canton repo.